overlord/snapstate/handlers.go

```diff
@@ -834,16 +834,10 @@ func (m *SnapManager) doPreDownloadSnap(t *state.Task, tomb *tomb.Tomb) error {
 	}
 
 	targetFn := snapsup.MountFile()
-	dlOpts := &store.DownloadOptions{
-		// pre-downloads are only triggered in auto-refreshes
-		Scheduled: true,
-		RateLimit: autoRefreshRateLimited(st),
-	}
-
```

Comment:
also not sure why this change?

Reply:
I think I removed it when the field was AutoRefresh, I'm not sure the scheduled header makes sense for the pre-download. In particular, since we now know that in other cases it's used to omit a change (which I assume would never apply to a download but still). Do you think I should leave it?

Reply:
it is still part of an auto-refresh, also we for sure want the ratelimit